Hub landing pages (#8713)
camunda/camunda-docs: docs/components/hub/workspace/index.md

---
id: workspace
title: Workspace
description: "Discover and use approved reusable assets, manage projects, and deliver business processes."
---

import DocsIcon from "@site/docs/components/assets/icon-docs.png";
import AoGrid from '../../react-components/\_ao-card';

Discover and use approved reusable assets, manage projects, and deliver business processes.

## About the workspace

With Camunda Hub, you'll collaborate in managed workspaces, discover and use approved catalog assets, and model and deploy business processes.

## Manage projects

> Develop and progress project releases through the stages of a typical development lifecycle:

Reviewer: ```suggestion
Develop project releases through the stages of a typical development lifecycle:
```
Sounds better like this to me

<AoGrid ao={[
{

> link: "../../modeler/web-modeler/process-applications",

Author: No, this is the correct link as validated locally. Adding the second "process-applications" gives a 404. 

According to [Docusaurus](https://docusaurus.io/docs/create-doc#doc-urls): 

> If a file is named [the same name as the parent folder], the file name won't be included in the URL.

title: "Create a project",
image: DocsIcon,
description: "Create a process application in your project to work on and deploy a set of related files.",
},
{
link: "../../modeler/web-modeler/process-applications/deploy-process-application",
title: "Deploy a project",
image: DocsIcon,
description: "Validate your process application in development before deploying it to testing, staging, or production.",
},
]} columns={2}/>

## Manage workspace settings

Manage your workspace's members, clusters, and general information:

<span class="link-arrow">[Get started](./manage-workspace/index.md)</span>

## Model business processes

Collaboratively design executable processes as the foundation for scalable IT and business automation:

<AoGrid ao={[
{
link: "../../modeler/web-modeler/modeling/model-your-first-diagram",
title: "Model your first diagram",
image: DocsIcon,
description: "Start modeling your first BPMN diagram.",
},
{
link: "../../modeler/web-modeler/run-or-publish-your-process",
title: "Run or publish your process",
image: DocsIcon,

> description: "Flexible options to run or publish a process in any environment and to any audience.",

Reviewer: ```suggestion
description: "Run or publish a process in any environment and for any audience.",
```
